guides/plugins/plugins/administration/add-new-tab.md

```diff
@@ -223,13 +223,14 @@ As shown in the previous example, your custom component is expected to be in a d
 
 ```text
 administration
-├── page
-│   └── sw-product-detail
+├── src
+│   └──page
 │       └── sw-product-detail
 │           ├── index.js
 │           └── sw-product-detail.html.twig
-├── view
-│   └── sw-product-detail-custom
+|── view
+│    └── sw-product-detail-custom
```

Comment:
There is no directory named `sw-product-detail-custom`. Hope that's just an example name.